python3: can't open file '/mnt/chromeos/removable/COLLECTIVE/github_sync/lucian_interactive.py': [Errno 2] No such file or directory
